### ISSUE TITLE
"amp does not work out-of-the-box with `F.binary_cross_entropy` or `torch.nn.BCELoss.`" Can we only use the two loss？

### ISSUE THREAD

**Issue by zs963048949** [NVIDIA/apex#1017]
The error message: **amp does not work out-of-the-box with `F.binary_cross_entropy` or `torch.nn.BCELoss.` It requires that the output of the previous function be already a FloatTensor.** 
I want to use the loss defined by myself,and more than one loss. In this case,can the apex work normally?

**Comment by sunpeng1996** [comment]
> The error message: **amp does not work out-of-the-box with `F.binary_cross_entropy` or `torch.nn.BCELoss.` It requires that the output of the previous function be already a FloatTensor.**
> I want to use the loss defined by myself,and more than one loss. In this case,can the apex work normally?
